front/js/script.js

```diff
@@ -0,0 +1,30 @@
+//The function below displays all products img, h3 and p on the home page
+
+fetch("http://localhost:3000/api/products")
+    .then(function(res) {
+        if (res.ok) {
+            return res.json();
+        }
+    })
+    .then(function(value) {
+            for (let i in value) {
+                const displayedProduct = document.getElementById("items");
+                const displayedProductArticle = document.createElement("article");
+                displayedProduct.appendChild(displayedProductArticle);
```

Comment:
Why do you add the dom element not completed to the visible dom? Why not happen displayedproductarticle only when you added the img and link title and description

Reply:
Reorganized the loop. All consts first, then applied attributes and finally dom additions. Smoothens page load up.